src/awkward/_v2/contents/content.py

```diff
@@ -588,6 +588,72 @@ def min(self, axis=-1, mask=True, keepdims=False, initial=None):
     def max(self, axis=-1, mask=True, keepdims=False, initial=None):
         return self._reduce(awkward._v2._reducers.Max(initial), axis, mask, keepdims)
 
+    def argsort(self, axis=-1, ascending=True, stable=False, kind=None, order=None):
+        """
+        Args:
+            axis (int): this argument defines an axis along
+                which to sort indices. The default is `-1`, which
+                sorts indices along the innermost axis. Axes are defined similar to
+                Numpy axes. `axis=0` is the "first" axis and `axis=-1` is
+                the "last" axis. Unlike other operations, sorting does not
+                support `axis=None`.
+            ascending (bool): if `True`, the elements are sorted in ascending
+                order.
+            stable (bool): if `True`, the order of equivalent elements is guaranteed
+                to be preserved.
+            kind ({‘quicksort’, ‘mergesort’, ‘heapsort’, ‘stable’}, optional):
+                an optional sorting algorithm. This is similar to NumPy
+                sort.
+            order (str or list of str, optional): similar to NumPy.
+
+        Puts indices of Content elements in an ordered sequence: numeric or lexicographical,
+        ascending or descending.
+
+        This function is similar to NumPy [sort](https://numpy.org/doc/stable/reference/generated/numpy.argsort.html).
+        """
```

Comment:
This isn't the high-level function, so it's not the place for the full docstring. We shouldn't be obligated to put such nice docstrings on all of these `Content` methods, so this will get removed eventually, or reduced to a minimal level of docstring that we're able to maintain on all of the `Content` methods uniformly. I'm going to leave it here for now because this information will be needed in the high-level function, when that gets updated.

Reply:
done